src/Backups/BackupCoordinationReplicatedAccess.h

```diff
@@ -47,7 +47,7 @@ class BackupCoordinationReplicatedAccess
 
     struct FilePathsAndHost
     {
-        std::unordered_set<String> file_paths;
+        std::set<String> file_paths;
```

Comment:
why does it need to be sorted?

Reply:
Because in [BackupCoordinationReplicatedAccess::getFilePaths](https://github.com/ClickHouse/ClickHouse/pull/69563/files#diff-267c93a46e11682a02f69d2fca27bbf21b172658ea906d71ad8a424dac7ac17eR44) we get the first `file_path` in this list and we want it to be the same on each host. The list of file paths contains the same elements on each host and if the list is sorted then we can be sure that the first element will be also the same.